contents/blog/web-analytics-mid-mortem.md

```diff
@@ -0,0 +1,75 @@
+---
+date: 2024-02-28
+title: "Mid-mortem: How is the web analytics beta going so far?"
+rootPage: /blog
+category: Inside PostHog
+author: ["joe-martin"]
+featuredImage: ../images/blog/web-analytics-mid-mortem.jpg
+featuredImageType: full
+---
+
+[Our web analytics beta](http://app.posthog.com/home#panel=feature-previews) started simple — a dashboard of some stats, plus the top site paths. We wanted to grow it into...
+
+1. A Google Analytics alternative..
+1. Which was low cost...
+1. Simple to use...
+1. Could be built quickly...
+1. And was suitable for technical founders...
+
+We didn’t put a lot of resources behind the idea. The ‘web analytics team’ is, in reality, just [Robbie](/handbook/small-teams/web-analytics). We think in situations like this where we can leverage existing work, one person learns faster than many — though it’s always worth looking at the progress you’re making. 
```

Comment:
I don't come away from this intro with a reason why, as a reader, I would want to read this piece. 

Maybe instead of introing what web analytics is, we can talk about lessons right away. An idea:

> Web analytics is a concept that has been around basically forever (in internet time). What more is there to learn when building it ourself? Turns out, a lot.
> Below the surface, building a web analytics product that users want requires solving complicated performance, pricing, and design decisions. To help give you insight into what we are thinking about (and maybe help you make similar decisions), we've listed them here.

Potentially helpful: https://posthog.com/product-engineers/measuring-feature-success

Reply:
I like this, thanks. Agree it's not a compelling opening. I'd want to keep the goal structure as I think that helps frame the conversation. How about...

> Web analytics as a concept has been around basically forever. What more is there to learn about how to build it? Turns out, a lot!
> 
> Build a product people want always means solving complicated performance, design, and pricing decisions in order to reach your goal. Our goal for web analytics was to build a Google Analytics alternative which was low cost, simple to use, could be built quickly, and was suitable for technical founders.
> 
> To give you insight into how we make decisions, let's explore the challenges under each part of that goal...